sparse/src/KokkosSparse_par_ilut.hpp

```diff
@@ -44,6 +44,17 @@ namespace Experimental {
   std::is_same<typename std::remove_const<A>::type, \
                typename std::remove_const<B>::type>::value
 
+/// @brief
```

Comment:
Thanks @e10harvey! You are working off a fork, so I can't make changes directly to the branch, but here is what I would put for the brief:

```
Performs the symbolic phase of par_ilut. The sparsity pattern of A will be analyzed and L_rowmap and U_rowmap will be populated with the L (lower triangular) and U (upper triagular) non-zero counts respectively. Having a separate symbolic phase allows for reuse when dealing with multiple matrices with the same sparsity pattern. This routine will set some values on handle for symbolic info (row count, nnz counts).

/// @tparam KernelHandle Template for the KernelHandle type
/// @tparam ARowMapType Template for A_rowmap type
/// @tparam AEntriesType Template for A_entries type
/// @tparam LRowMapType Template for L_rowmap type
/// @tparam URowMapType Template for U_rowmap type
/// @param handle The kernel handle. It is expected that create_par_ilut_handle has been called on it
/// @param A_rowmap The row map (row nnz offsets) for the A CSR (Input)
/// @param A_entries The entries (column ids) for the A CSR (Input)
/// @param L_rowmap The row map for the L CSR, should already be sized correctly (numRows+1) (Output)
/// @param U_rowmap The row map for the U CSR, should already be sized correctly (numRows+1) (Output)
```

Reply:
@jgfouca I don't know the details of the underlying algorithm.  Would it be accurate to say that the symbolic phase determines the graphs (sparsity patterns) of the $L$ and $U$ factors?

Reply:
@jhux2 , yes that's right. I think the idea is that there may be a use case where the user has multiple matrices with different values but the same sparsity pattern, so you can reuse the symbolic phase for all of them.